gef.py

```diff
@@ -8851,6 +8857,16 @@ def do_invoke(self, argv):
             err("Unmapped address")
             return
 
+        ref_addr = safe_parse_and_eval(reference)
+        if ref_addr is None:
+            err("Invalid reference")
+            return
+
+        ref_addr = int(ref_addr)
+        if process_lookup_address(ref_addr) is None:
+            err("Unmapped reference")
```

Comment:
Let's add the attempted ref to the error

Reply:
I am actually wondering whether the reference address must be mapped or if that doesn't matter and this check can be omitted? certainly in most cases it will/should be mapped but I don't see a specific reason to forbid using an unmapped address here just as a reference for the offset. What do you think?